Surface_mesh/include/CGAL/Surface_mesh/Surface_mesh.h

```diff
@@ -44,6 +45,7 @@
 #include <CGAL/boost/graph/graph_traits_Surface_mesh.h>
 #include <CGAL/boost/graph/iterator.h>
 #include <CGAL/boost/graph/Euler_operations.h>
+#include <CGAL/IO/File_scanner_OFF.h>
 
```

Comment:
Why `#include <boost/fusion/container/map.hpp>` and `#include <CGAL/IO/File_scanner_OFF.h>`?